.drone.yml

```diff
@@ -189,7 +189,7 @@ steps:
 
   - name: ui-tests
     # see: docker/dockerfiles/Dockerfile
-    image: codedotorg/code-dot-org:1.9.4
+    image: codedotorg/code-dot-org:pycall-pdm
```

Comment:
Need to push a new image after this is merged to go back to version numbering (and/or merge my docker refactor pr)

Reply:
💯, got it on the checklist